setup.py

```diff
@@ -128,6 +128,7 @@
     "torchvision",
     "transformers>=4.25.1",
     "urllib3<=2.0.0",
+    "peft>=0.5.0"
```

Comment:
```suggestion
```

Should we for now maybe just remove it from the `setup.py` to not confuse people? It's a soft-dependency anyways and we can add it once a new version comes out no? The other option would be to go this way maybe: https://stackoverflow.com/questions/32688688/how-to-write-setup-py-to-include-a-git-repository-as-a-dependency

Reply:
We can just add a `pip install git+https://github.com/huggingface/peft` to the CI